core/src/main/kotlin/ch/ergon/dope/DopeQuery.kt

```diff
@@ -1,3 +1,11 @@
 package ch.ergon.dope
 
-data class DopeQuery(val queryString: String, val parameters: Map<String, Any>)
+import ch.ergon.dope.resolvable.expression.unaliased.type.ParameterManager
+import ch.ergon.dope.resolvable.expression.unaliased.type.collection.IteratorManager
+
+data class DopeQuery(val queryString: String, val parameters: Map<String, Any>, val manager: DopeQueryManager)
```

Comment:
Why do we have to store the manager in the DopeQuery? According to my understanding, the manager is only needed in the `.toDopeType(manager)` function?